### PR TITLE
testing/py-geoip-api && geoip-database: new aports

### PR DESCRIPTION
**Pull request by itoffshore**
* testing/py-geoip-api
* testing/geoip-database

### CODE REVIEW

File: testing/geoip-database/APKBUILD [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+pkgver=20170714
+pkgrel=0
+pkgdesc="GeoIP library (legacy country databases)"
+url="http://maxmind.com"
+arch="noarch"
+license="CC-4.0"
+depends=""
+options="!check"
+source="http://geolite.maxmind.com/download/geoip/database/GeoLiteCountry/GeoIP.dat.gz
+	http://geolite.maxmind.com/download/geoip/database/GeoIPv6.dat.gz
```
Comment (jirutka) [inline testing/geoip-database/APKBUILD]: Once again, this is not acceptable. If upstream does not provide stable source, then you must use `snapshot()`.

File: testing/geoip-database/APKBUILD [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+	"
+builddir="$srcdir/"
+
+build() {
+	gunzip "$builddir"/*.gz
```
Comment (jirutka) [inline testing/geoip-database/APKBUILD]: This is not “build”…

File: testing/py-geoip-api/APKBUILD [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+	local python="$1"
+	pkgdesc="$pkgdesc (for $python)"
+	depends="$depends $python"
+	install_if="$pkgname=$pkgver-r$pkgrel $python"
+
+	cd "$builddir"
+	$python setup.py install --prefix=/usr --root="$subpkgdir"
+}
+
+check() {
```
Comment (jirutka) [inline testing/py-geoip-api/APKBUILD]: Check phase is run after _build_ and before _package_, so please put it between these functions to make abuild more understandable.

**Comment by itoffshore** [comment]
s/be fixed now

**Comment by itoffshore** [comment]
moved to https://github.com/alpinelinux/aports/pull/1907

**Pull request closed without merging**